includes/class-wcs-change-payment-method-admin.php

```diff
@@ -91,12 +90,24 @@ public static function display_fields( $subscription ) {
 				}
 
 				echo '</div>';
-
 			}
 		}
 
-		wp_nonce_field( 'wcs_change_payment_method_admin', '_wcsnonce' );
+		// If the subscription has a payment method but has turned off auto-renewal, display a checkbox so automatic renewals can be resumed.
+		if ( $subscription->get_requires_manual_renewal() && ! empty( $payment_method ) && ! wcs_is_manual_renewal_required() ) {
+			echo '<p class="form-field form-field-wide">';
+			woocommerce_form_field(
+				'wc-subscription-auto-renew',
+				[
+					'type'    => 'checkbox',
+					'label'   => esc_html__( 'Auto renew', 'woocommerce-subscriptions' ) . wcs_help_tip( __( 'This subscription has automatic renewals turned off. Use this option to turn them on.', 'woocommerce-subscriptions' ) ),
```

Comment:
This line includes the help tip which looks like this

<img width="400" alt="Screenshot 2023-10-31 at 1 38 07 pm" src="https://github.com/Automattic/woocommerce-subscriptions-core/assets/8490476/048262d0-4461-4ded-a1be-72ef8dc6d80d">